packages/create-sitecore-jss/src/index.ts

```diff
@@ -0,0 +1,77 @@
+#!/usr/bin/env node
+import fs from 'fs';
+import path from 'path';
+import { prompt } from 'inquirer';
+import { initRunner } from './init-runner';
+import parseArgs, { ParsedArgs } from 'minimist';
+import { getBaseTemplates } from './common/utils/helpers';
+
+// parse any command line arguments passed into `init sitecore-jss`
+// to pass to the generator prompts and skip them.
+// useful for CI and testing purposes
+const argv: ParsedArgs = parseArgs(process.argv.slice(2), { boolean: ['appPrefix', 'yes'] });
+
+const main = async () => {
+  let templates: string[] = [];
+
+  // check if templates were provided
+  if (argv._.length > 0 && argv._[0] !== undefined) {
+    // use positional parameter
+    templates = [argv._[0]];
+  } else {
+    // use --templates arg
+    templates = argv.templates?.trim().split(',') || [];
+  }
+  const baseTemplates = await getBaseTemplates(path.resolve(__dirname, 'templates'));
+  // validate/gather templates
+  if (!templates.length) {
+    const answer = await prompt({
+      type: 'list',
+      name: 'template',
+      // eslint-disable-next-line quotes
+      message: "Select the template you'd like to create?",
+      choices: baseTemplates,
+      default: 'nextjs',
+    });
+    templates.push(answer.template);
+  }
+
+  // validate/gather destination
+  const defaultDestination = `${process.cwd()}${
+    argv.appName ? '\\' + argv.appName : '\\sitecore-jss-app'
```

Comment:
Instead of "sitecore-jss-app", use `templates[0]`? This would be in line with jss create experience.